pkg/noderesourcetopology/filter_test.go

```diff
@@ -553,6 +586,221 @@ func TestNodeResourceTopologyMultiContainerPodScope(t *testing.T) {
 	}
 }
 
+func TestNodeResourceTopologyMultiContainerContainerScope(t *testing.T) {
+	nodeTopologies := []*topologyv1alpha1.NodeResourceTopology{
```

Comment:
1. It is fine to have _some_ code duplication, especially in test code - https://evgenii.com/blog/code-duplication-in-unit-tests/ 

2. If we would like to change some of the values in the NRT object, it will involve changing the values for the second method as well. Are we sure we want to have that?